packages/perseus/src/widgets/__tests__/radio/base-radio_test.jsx

```diff
@@ -79,4 +79,87 @@ describe("base-radio", () => {
             });
         });
     });
+
+    describe("selecting and deselecting options", () => {
+        it("deselects multi-select choices", () => {
+            // Arrange
+            const apiOptions: APIOptions = {
+                styling: {radioStyleVersion: "final"},
+            };
+            let updatedValues = null;
+            const onChangeHandler = (newValues) => {
+                updatedValues = newValues;
+            };
+
+            render(
+                <BaseRadio
+                    multipleSelect={true}
+                    countChoices={false}
+                    numCorrect={1}
+                    editMode={false}
+                    labelWrap={false}
+                    apiOptions={apiOptions}
+                    choices={[
+                        {content: "Option 1", correct: false, checked: false},
+                        {content: "Option B", correct: false, checked: false},
+                        {content: "Option Gamma", correct: true, checked: true},
+                        {
+                            content: "Option Delta",
+                            correct: false,
+                            checked: false,
+                        },
+                    ]}
+                    onChange={onChangeHandler}
+                />,
+            );
+
+            // Act
+            userEvent.click(
+                screen.getByText("(Choice C, Checked)", {selector: "div"}),
```

Comment:
I've had success with `getByRole` in the radio widget (my PRs should have examples) and that's the preferred way to get interactive elements.

Reply:
Updated! Thanks for the recommendation :)